src/utils/gSheetDataFetcher.ts

```diff
@@ -13,7 +11,16 @@ export const gSheetDataFetcher = async (
   spreadsheetID: string,
   sheetName: string
 ) => {
-  const jwtClient = await authorize()
+  let jwtClient
+  try {
+    jwtClient = await authorize()
+  } catch (e) {
+    console.dir(e.message)
+    throw new Error(
+      "Error generating JWT client for google - ensure your GOOGLE_CLIENT_SECRET is a single string with `\\n` for each linebreak. See console output for full error message."
```

Comment:
I'm not familiar with typescript backslash-escaping. Will this print 2 backslashes in the error message or 1?

Reply:
```
~/Projects/work/kaws [GROW-1540 ✅]
myk@mykbook😙 $ node
> console.log('this is a test of `\\n` how this will log out.')
this is a test of `\n` how this will log out.
```